examples/freely_decaying_mediterraneum.jl

```diff
@@ -0,0 +1,119 @@
+using GLMakie
+using Oceananigans
+using Oceananigans: architecture
+using Oceananigans.Utils: prettytime
+using ClimaOcean.Bathymetry: regrid_bathymetry
+using ClimaOcean.ECCO2: ecco2_field, ecco2_center_mask
+using ClimaOcean.VerticalGrids: stretched_vertical_faces, PowerLawStretching
+using ClimaOcean.InitialConditions: three_dimensional_regrid!, adjust_tracers!
```

Comment:
Should we start exporting all the things we need so we don't have to have these big `using` statements at the top?